### ISSUE TITLE
Blin should automatically create useful content for github tickets

### ISSUE THREAD

**Issue by AlexDaniel** [Raku/Blin#22]
Currently, tickets like these are created manually: https://github.com/rakudo/rakudo/issues/3553 (more examples can be found [here](https://github.com/rakudo/rakudo/issues?q=blin+is%3Aissue+sort%3Aupdated-desc+is%3Aclosed)).

Discussion with @Altai-man: https://colabti.org/irclogger/irclogger_log/raku-dev?date=2020-03-22#l44

Another discussion with more info: https://colabti.org/irclogger/irclogger_log/raku-dev?date=2020-03-23#l15

Basically, Blin generates a huge `overview` file and another even bigger json file, but both can't be used directly as github tickets. This forces release managers to create ticket text manually, which takes minutes (and this is repeated multiple times per release). There's no need to make it automatically create a GitHub ticket via API, just generating content for a ticket is enough.

**Comment by AlexDaniel** [comment]
This was resolved by @Xliff++ in https://github.com/Raku/Blin/pull/23
